### PR TITLE
fix(TUP-21075)Problem of display in the tMap with Mojave

### PR DESCRIPTION
**Pull request by jding-tlnd**
https://jira.talendforge.org/browse/TUP-21075

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java [hunk 1]
```diff
@@ -158,8 +161,20 @@ public void handleEvent(ExtendedButtonEvent event) {
 
         };
 
+        final List<InputTable> inputTables = mapperManager.getInputTables();
```
Comment (cmeng-talend) [inline main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java]: Why don't put the inputTables into the handleEvent function? and add NPE check?

File: main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java [hunk 2]
```diff
@@ -197,8 +212,20 @@ public void handleEvent(ExtendedButtonEvent event) {
 
         };
 
+        final List<OutputTable> outputTables = mapperManager.getOutputTables();
```
Comment (cmeng-talend) [inline main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java]: Why don't put the outputTables into the handleEvent function? and add NPE check?

File: main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java (hunk 1 above)
Comment (jding-tlnd) [reply main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java]: Solved, thanks.
And this mapperManager here shouldn't be null, this TabFolderEditors defined from UIManager  defined from MapperManager. if the mapperManger is null here, the mapper window wound not display normally.

File: main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java (hunk 2 above)
Comment (jding-tlnd) [reply main/plugins/org.talend.designer.mapper/src/main/java/org/talend/designer/mapper/ui/tabs/TabFolderEditors.java]: Solved, thanks.
And this mapperManager here shouldn't be null, this TabFolderEditors defined from UIManager  defined from MapperManager. if the mapperManger is null here, the mapper window wound not display normally.

**Comment by jding-tlnd** [comment]
Sorry for the comment of second commit,it's wrong, it's handwriting error by my careless.

**Pull request merged**